pkg/controllers/sync/controller.go

```diff
@@ -326,55 +331,69 @@ func (s *SyncController) HasSynced() bool {
 		return false
```

Comment:
Why do we log for `ftcManager` and `fedInformerManager` but not `fedAccessor`?

Reply:
The comment above explains it.